drivers/dide/R/bootstrap.R

```diff
@@ -8,9 +8,9 @@ bootstrap_path_windows <- function(development) {
 
 bootstrap_path_linux <- function(development) {
   if (!is.null(development) && !isFALSE(development)) {
-    "/wpia-hn/Hipercow/bootstrap-dev-linux"
+    "/mnt/cluster/Hipercow/bootstrap-dev-linux"
   } else {
-    "/wpia-hn/Hipercow/bootstrap-linux"
+    "/mnt/cluster/Hipercow/bootstrap-linux"
   }
```

Comment:
Just because we are mounting the wpia-hn cluster share in a different place in linux now - /wpia-hn is now /mnt/cluster This is where hipercow finds the R packages that it needs itself to function on the cluster node.